Unit tests fail on Windows for backwards vs forward slashes difference

Another fun Windows exclusive bug :)

```diff
1) TSLintRunner
       runTsLint
         should return an error for test file:

      AssertionError: expected 'C:/Code/typescript-tslint-plugin/test-data/with-tslint/test.ts' to equal 'C:\\Code\\typescript-tslint-plugin\\test-data\\with-tslint\\test.ts'
      + expected - actual

      -C:/Code/typescript-tslint-plugin/test-data/with-tslint/test.ts
      +C:\Code\typescript-tslint-plugin\test-data\with-tslint\test.ts

      at Context.it.only (out\test\runner.test.js:29:58)
```

--- a/src/runner/test/runner.test.ts
+++ b/src/runner/test/runner.test.ts
@@ -31,7 +31,7 @@ describe('TSLintRunner', () => {
             expect(result.lintResult.warningCount).to.equal(0);
 
             const firstFailure = result.lintResult.failures[0];
-            expect(firstFailure.getFileName()).to.equal(filePath);
+            expect(path.normalize(firstFailure.getFileName())).to.equal(filePath);
             expect(firstFailure.getRuleName()).to.equal('array-type');
 
             const fix = firstFailure.getFix();

